boards/nxp/frdm_mcxn947/frdm_mcxn947.dtsi

```diff
@@ -136,3 +136,25 @@
 	pinctrl-0 = <&pinmux_dac0>;
 	pinctrl-names = "default";
 };
+
+&enet {
+	status = "okay";
+	pinctrl-0 = <&pinmux_enet_qos>;
+	pinctrl-names = "default";
+};
+
+&enet_mac {
+	phy-connection-type = "rmii";
+	zephyr,random-mac-address;
+	status = "okay";
+	phy-handle = <&phy>;
+};
+
+&enet_mdio {
+	status = "okay";
+	phy: ethernet-phy@0 {
+		compatible = "ethernet-phy";
+		reg = <0>;
+		status = "okay";
+	};
+};
```

Comment:
Can these nodes be set to `"status = okay"` in the _cpu0 file. It gives flexibility in selecting which peripherals would be needed for cpu1 when we add that support.